Packs/McAfeeNSM/Integrations/McAfeeNSMv2/McAfeeNSMv2.py

```diff
@@ -0,0 +1,1939 @@
+    alert_det['Target'] = alert_det.get('summary', {}).get('target')
+    alert_det['MalwareFile'] = alert_det.get('details', {}).get('malwareFile')
+    alert_det['Details'] = alert_det.get('details')
+    details_obj = alert_det.get('details', {})
+    summary_obj = alert_det.get('summary', {})
+    description_obj = alert_det.get('description', {})
+    reference_obj = alert_det.get('description', {}).get('reference', {})
+    del details_obj['malwareFile']
+    del summary_obj['target']
+    del summary_obj['attacker']
+    del summary_obj['event']
+    del description_obj['protocols']
+    del description_obj['attackSubCategory']
+    del description_obj['attackCategory']
+    del description_obj['definition']
+    del alert_det['assignTo']
+    del alert_det['alertState']
+    del alert_det['name']
+    del alert_det['details']
+    del reference_obj['nspId']
+    return alert_det
+
+
+def get_addresses_from_response(response: Dict) -> List:
+    """ Returns the addresses from the response, for the human-readable in the command get_rule_object.
+        Args:
+            response: Dict - The response from the API.
+        Returns:
+            The list of addresses.
+    """
+    rule_type = response.get('ruleobjType', '')
+    number = 4 if '4' in rule_type else 6
+    if 'HOST' in rule_type:
+        return response.get(f'HostIPv{number}', {}).get(f'hostIPv{number}AddressList', [])
+    elif 'ADDRESS_RANGE' in rule_type:
+        return response.get(f'IPv{number}AddressRange', {}).get(f'IPV{number}RangeList', [Dict])
+    else:  # 'NETWORK'
+        return response.get(f'Network_IPV_{number}', {}).get(f'networkIPV{number}List', [])
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client: Client, username_n_password: str) -> str:
+    """ Test the connection to McAfee NSM.
+    Args:
+        client: Client - A McAfeeNSM client.
+        username_n_password: str - The string that contains username:password to be encoded.
+    Returns:
+        'ok' if the connection was successful, else throws exception.
+    """
+    try:
+        get_session(client, username_n_password)
+        return 'ok'
+    except DemistoException as e:
+        raise Exception(e.message)
+
+
+def list_domain_firewall_policy_command(client: Client, args: Dict) -> CommandResults:
+    """ Gets the list of Firewall Policies defined in a particular domain.
+    Args:
+        client: client - A McAfeeNSM client.
+        args: Dict - The function arguments.
+    Returns:
+        A CommandResult object with the list of Firewall Policies defined in a particular domain.
+    """
+    domain_id = arg_to_number(args.get('domain_id', None))
+    limit = arg_to_number(args.get('limit', 50)) or 50
+    page = arg_to_number(args.get('page'))
+    page_size = arg_to_number(args.get('page_size'))
+    if (page and not page_size) or (not page and page_size):
+        raise Exception('If you enter one of the parameters page or page_size, you have to enter both.')
```

Comment:
```suggestion
        raise Exception('Please provide both page and page_size arguments.')
```

Reply:
Maybe worth adding something like this also:
    # verify that only one of the arguments is given
    if limit and (page or page_size):
        return 'Please provide either limit argument or page and page_size arguments.'